handlers/conceptHandlers.go

```diff
@@ -29,7 +30,22 @@ func GetConceptUpdateHandler() func(c *gin.Context) {
 		updateId := c.Param("updateId")
 		logrus.Infof("Get update %s for concept %s", updateId, conceptId)
 
-		concept, status := service.GetConceptUpdate(c.Request.Context(), conceptId, updateId)
+		// Get the skip and limit values from the query parameters
+		skip := c.Query("skip")
+		limit := c.Query("limit")
+
+		// Convert the skip and limit values to integers
+		skipInt, err := strconv.Atoi(skip)
+		if err != nil {
+			skipInt = 0
+		}
+
+		limitInt, err := strconv.Atoi(limit)
+		if err != nil {
+			limitInt = 10
+		}
+
+		concept, status := service.GetConceptUpdate(c.Request.Context(), conceptId, updateId, skipInt, limitInt)
```

Comment:
Fjern og flytt det i stedet opp til `service.GetConceptUpdates`
```suggestion
		concept, status := service.GetConceptUpdate(c.Request.Context(), conceptId, updateId)
```